.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,11 +1,11 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Support Request
+  - name: Access paid support
     url: https://sentry.io/support
     about: Use our dedicated support channel for paid accounts.
-  - name: Ask a question about self-hosted Sentry (aka onpremise)
+  - name: Discuss self-hosted
     url: https://github.com/getsentry/self-hosted/issues/new
     about: Please use the `self-hosted` repo for questions about self-hosted Sentry.
-  - name: Report a security vulnerability
-    url: https://sentry.io/security/#vulnerability-disclosure
-    about: Please see our guide for responsible disclosure.
+  - name: Bypass issue templates
+    url: https://github.com/getsentry/sentry/issues/new
```

Comment:
iirc this link is only clickable for repo admins

Reply:
Confirmed with @souredoutlook that Write role-holders are also not redirected. 👍